centovacast.php

```diff
@@ -1826,13 +1831,37 @@ private function getFieldsFromInput(array $vars, $package)
         ];
 
         $fields = array_merge((array) $package->meta, $fields);
-
-        if ($fields['diskquota'] == 0) {
-            $fields['diskquota'] = 'unlimited';
-        }
-        if ($fields['transferlimit'] == 0) {
-            $fields['transferlimit'] = 'unlimited';
-        }
+		
+		//configoptions (Configurable Options)
+		$fieldsConfigOptions = array_merge((array) $package->meta, $fields['configoptions']);
+		
+		
+		//If centovacast_diskquota configoptions is specified
```

Comment:
This comment is inaccurate.  diskquota is being checked, not centovacast_diskquota